docs-source/content/faq/external-clients.md

```diff
@@ -0,0 +1,213 @@
+
+#### Federation
+
+- Deploy the remote clients inside another Kubernetes cluster.
+
+- Federate the two clusters so that they effectively share the same DNS namespace. See [Federation](https://kubernetes.io/docs/concepts/cluster-administration/federation/) in the Kubernetes documentation.  Note that federation is a relatively new Kubernetes feature as of 2019.
+
+- If the addresses (URLs) for remote access from the client cluster to the original cluster are not the same as the addresses used by clients running within the original cluster itself, then it is often necessary to configure a custom channel for the T3 protocol in the original cluster.  See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
+
+
+#### Load Balancer Tunneling
+
+
+- Configure a custom channel for the T3 protocol in WebLogic that (a) enables HTTP Tunneling, and (b) specifies an external address and port that correspond to the address and port remote clients will use access the load balancer.  See  [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
+
+- Setup a load balancer that redirects HTTP traffic to the custom channel. For a discussion of load balancers see [Ingress]({{<relref "/userguide/managing-domains/ingress/_index.md">}}).
+
+- Remote clients can then access the custom channel using an 'http://' URL instead of a 't3://' URL.
+
+- Review the [Security Notes](#security-notes) below.
+
+#### Kubernetes Node Ports
+
+- Configure a custom channel for the T3 protocol in WebLogic that specifies an external address and port that are suitable for remote client use.  See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
+
+- Deploy a Kubernetes NodePort to publicly expose the WebLogic ports. See [Setting up a NodePort](#setting-up-a-nodeport).
+
+- Review the [Security Notes](#security-notes) below.
+
+### Adding a WebLogic Custom Channel
+
+#### When is a WebLogic Custom Channel needed?
+
+WebLogic implicitly creates a multi-protocol default network channel that spans the Listen Address and Port fields specified on each server in the cluster. Here are some reasons you may need to configure an additional dedicated WebLogic custom channel to handle remote EJB or JMS client network traffic:
+
+- A default channel does not provide a way to configure an external address and port, but external listen address and/or port configuration is needed when a channel's configured listen address and/or port would not work if used to form a URL in the remote client.
+  - This is because remote EJB and JMS clients internally use their client's channel's configured network information to reconnect to WebLogic when needed. (The EJB and JMS clients do not always use the initial URL specified in the client's JNDI context.)
```

Comment:
feels like this would be better as prose than a list

Reply:
ok - done